MusicShop/src/main/java/ua/lviv/iot/musicshop/manager/TextRegex.java

```diff
@@ -0,0 +1,28 @@
+package ua.lviv.iot.musicshop.manager;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class TextRegex {
+
+  public String textEdit(String text, int length) {
```

Comment:
Please provide a more clear method signature. It should be as self-descriptive as possible -- the person who reads your code should be able to understand the context when they see the method signature, e. g. what kind of editing does this method do? What is `length`?
This is especially important for RegEx tasks, as I can't be sure you've done it right if I don't understand what the original task was.